sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -1580,22 +1580,37 @@ object functions {
   //////////////////////////////////////////////////////////////////////////////////////////////
 
   /**
-   * Computes the length of a given string value
-   *
+   * Computes the length of a given string value.
+   * 
```

Comment:
Sure, but is there a benefit? The `StringUtils` implementation does look reasonable, doesn't it? Do you have a special concern about their implementation? (http://grepcode.com/file/repo1.maven.org/maven2/org.apache.commons/commons-lang3/3.0/org/apache/commons/lang3/StringUtils.java#StringUtils.getLevenshteinDistance%28java.lang.CharSequence%2Cjava.lang.CharSequence%29)